syncd/syncd.cpp

```diff
@@ -2782,7 +2782,6 @@ void processFlexCounterGroupEvent(
 {
     SWSS_LOG_ENTER();
 
-    std::lock_guard<std::mutex> lock(g_mutex);
```

Comment:
can''t be removed, consumer must be under lock

Reply:
Is it because of the processNotification thread in syncd/syncd_notifications.cpp?